tests/functional/api/test_query.py

```diff
@@ -7,3 +11,28 @@ def test_basic_query(eth_tester_provider):
     df = chain.blocks.query("number", "timestamp")
     assert len(df) == 4
     assert df["timestamp"][3] > df["timestamp"][2] >= df["timestamp"][1] >= df["timestamp"][0]
+    df_all = chain.blocks.query("*")
+    columns = list(df_all.columns)
+    assert [
+        "gas_data",
+        "consensus_data",
+        "hash",
+        "number",
+        "parent_hash",
+        "size",
+        "timestamp",
+    ] == columns
+
+
+def test_block_query(eth_tester_provider):
+    chain.mine(3)
+    with pytest.raises(ValidationError) as err:
+        BlockQuery(columns=["columns"], start_block=0, stop_block=2)
+    assert "Unrecognized field 'columns" in str(err.value)
```

Comment:
Might be nicer to show a typo:
```suggestion
        BlockQuery(columns=["numbr"], start_block=0, stop_block=2)
    assert "Unrecognized field 'numbr'" in str(err.value)
```